e2e/pageobjects/ide/Editor.ts

```diff
@@ -296,6 +298,39 @@ export class Editor {
         return lineYCoordinate;
     }
 
+    async clickOnTheTextInDefinedPosition(line: number, column: number) {
+        const additionalShiftingToY = 19;
+        const additionalShiftingToX = 1;
+        const yPosition: number = await this.getLineYCoordinates(line) + additionalShiftingToY;
+        const xPosition: number = column + additionalShiftingToX;
+        new  ActionSequence(this.driverHelper.getDriver()).
+        mouseMove({x: xPosition, y: yPosition}).
+        click().
+        perform();
+}
+
+    async goToDefinitionWithMouseClicking(line: number, column: number) {
+        const additionalShiftingToY = 19;
+        const additionalShiftingToX = 1;
+        const yPosition: number = await this.getLineYCoordinates(line) + additionalShiftingToY;
+        new  ActionSequence(this.driverHelper.getDriver()).
+        keyDown(Key.CONTROL).
+        mouseMove({x: column + additionalShiftingToX, y: yPosition}).
+        click().
+        keyDown(Key.CONTROL).
+        perform();
+    }
+
+    async mouseContextClickInDefinedPosition(line: number, column: number) {
```

Comment:
`mouseContextClickInDefinedPosition` is to complex name.
What about `mouseRightButtonClick()`?

Reply:
renamed